docs/compilers/CSharp/CommandLine.md

```diff
@@ -41,7 +41,7 @@
 | **ERRORS AND WARNINGS**
 | `/warnaserror`{`+`&#124;`-`} | Report all warnings as errors
 | `/warnaserror`{`+`&#124;`-`}`:`*warn list* | Report specific warnings as errors
-| `/warn`:*n* | Set warning level (0-4) (Short form: `/w`)
+| `/warn`:*n* | Set warning level (0-5) (Short form: `/w`)
```

Comment:
It actually seem that any non-negative value, even if bigger than the warning level range, will be valid.

https://github.com/dotnet/roslyn/blob/a6013f3213c902c0973b2d371c3007217d610533/docs/compilers/CSharp/Warnversion%20Warning%20Waves.md

> If you want the compiler to produce all applicable warnings, you can specify /warn:9999.

Should I add a note similar to:

```
Any value larger than the valid range will be assumed as the largest possible value in the range.
```